fixtures/webview.py

```diff
@@ -85,3 +85,22 @@ def webview_instance(webview_base_url):
         return url.subdomain
     else:
         return "prod"
+
+
+@pytest.fixture
+def redirecting_books_titles():
+    """Returns all the books from the list of redirecting books to rex
+    """
+
+    here = os.path.join(DATA_DIR, "redirected_books.txt")
```

Comment:
You don't need this. At the top of the file is a global called `DATA_DIR`. Please use other examples to your advantage.